src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -4693,8 +4689,8 @@ public static void StartsWith_NullInStrings(StringComparison comparison)
         {
             Assert.False("\0test".StartsWith("test", comparison));
             Assert.False("te\0st".StartsWith("test", comparison));
-            Assert.True("te\0st".StartsWith("te\0s", comparison));
-            Assert.True("test\0".StartsWith("test", comparison));
+            Assert.StartsWith("te\0s", "te\0st", comparison);
+            Assert.StartsWith("test", "test\0", comparison);
```

Comment:
😟 I worry about changing these tests since it no longer guarantees we are using specific APIs.

Reply:
Oops. will fix... I think I have made similar changes for `EndsWith` as well.